test/commands/add/alias_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require "./test/helper"
+
+clean_describe "add alias" do
+  subject { run_cmd("add alias #{location_name} #{nickname}") }
+
+  let(:content) { CONTENT }
+
+  describe "when location name and alias are blank" do
+    let(:location_name) { nil }
+    let(:nickname) { nil }
+
+    it "prints an error message" do
+      stderr_only 'Error: Expected "[Location Name]" "[Alias]"'
+    end
+  end
+
+  describe "when location name has no matches" do
+    let(:location_name) { "Garbage" }
+    let(:nickname) { "Big Apple Pie" }
+
+    it "prints an error message" do
+      stderr_only 'Error: No location found for "Garbage"'
+    end
+  end
+
+  describe "when location alias has more than one match" do
+    let(:location_name) { "'New York City'" }
+    let(:nickname) { "'Big Apple'" }
+    before do
+      run_cmd("add location Manhattan")
+      run_cmd("add alias Manhattan 'Big Apple'")
+    end
+
+    it "prints an error message" do
+      stderr_only 'Error: The location alias "Big Apple" is already taken by "Manhattan"'
+    end
+  end
+
+  describe "when location name has one match" do
+    let(:location_name) { "'New York City'" }
+
+    describe "when alias is blank" do
```

Comment:
Would you mind adding another test for when `nickname` is just `nil`? That will I think capture the `friends add nickname 'New York City'` example that I want to make sure we can handle.